package.json

```diff
@@ -19,7 +19,7 @@
     "test": "mocha node-tests && ember test"
   },
   "dependencies": {
-    "babel-plugin-debug-macros": "^0.1.11",
+    "babel-plugin-debug-macros": "^0.2.0-beta.6",
```

Comment:
This reminds me: I should release 0.2.0 stable...

Reply:
If you want to cut that soon (now) I'll bump to that version instead of the beta.